### PR TITLE
Add jetty.hostname property to properly generate email links within Azkaban email notifications (even on tomcat)

### PR DESCRIPTION
**Pull request by sengelbert**
Simple change to add jetty.hostname property for correct email url links.

### CODE REVIEW

File: azkaban-soloserver/src/package/conf/azkaban.properties [hunk 1]
```diff
@@ -24,6 +24,7 @@ h2.create.tables=true
 velocity.dev.mode=false
 
 # Azkaban Jetty server properties. Ignored in tomcat
+jetty.hostname=localhost
```
Comment (juhoautio) [inline azkaban-soloserver/src/package/conf/azkaban.properties]: Could you update the comment to say that even on tomcat the jetty.* properties are used to generate links for email notifications?

File: azkaban-webserver/src/package/conf/azkaban.properties [hunk 2]
```diff
@@ -26,6 +26,7 @@ mysql.numconnections=100
 velocity.dev.mode=false
 
 # Azkaban Jetty server properties.
+jetty.hostname=localhost
```
Comment (juhoautio) [inline azkaban-webserver/src/package/conf/azkaban.properties]: Could you update the comment to say that even on tomcat the jetty.* properties are used to generate links for email notifications?

**Comment by juhoautio** [comment]
:+1:

File: azkaban-soloserver/src/package/conf/azkaban.properties (hunk 1 above)
Comment (sengelbert) [inline azkaban-soloserver/src/package/conf/azkaban.properties]: Updated the PR. Git commit comments cannot be changed as far as I know.

Comment (juhoautio) [inline azkaban-soloserver/src/package/conf/azkaban.properties]: Sorry, but I didn't mean the PR title. I mean the comments in the `azkaban.properties` files. For example "# Azkaban Jetty server properties. Ignored in tomcat" above ^

Comment (juhoautio) [inline azkaban-soloserver/src/package/conf/azkaban.properties]: The original PR title was fine I would say, LOL :)

**Comment by HappyRay** [comment]
The property files have changed.
It's a fine idea to add the config to the solo-server conf/azkaban.properties as an example. 

Are you still interested in this PR?

**Comment by HappyRay** [comment]
Please re-opne if you are interested in continuing this PR.

**Pull request closed without merging**